scripts/test.sh

```
@@ -505,13 +528,19 @@
 
 ' > main.rs
 
-cargo run &  # Start server in background
+cargo run --features with-wynd &  # Start server in background
 SERVER_PID=$!  # Store server process ID
 
-sleep 20
+for i in {1..60}; do
```

Shellcheck (0.10.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)